packages/flutter/lib/src/material/text_field.dart

```diff
@@ -681,6 +691,8 @@ class TextField extends StatefulWidget {
     properties.add(DiagnosticsProperty<bool>('autofocus', autofocus, defaultValue: false));
     properties.add(DiagnosticsProperty<bool>('obscureText', obscureText, defaultValue: false));
     properties.add(DiagnosticsProperty<bool>('autocorrect', autocorrect, defaultValue: true));
+    properties.add(DiagnosticsProperty<bool>('enableSmartDashes', enableSmartDashes, defaultValue: true));
```

Comment:
The defaultValue seems to be `!obsecureText` in the constructor.  Is there a reason for the defaultValue to deviate from the constructor?

Reply:
There is not, good catch.  I'll change it to `!obscureText`.